docs/distributions.html

```diff
@@ -0,0 +1,369 @@
+<li><a href='configuration.html'>Configuration</a></li>
+<li><a href='docker.html'>Docker</a></li>
+<li><a href='update.html'>Update</a></li>
+</ul>
+<li class='sub--on'>
+<h2>Digging Deeper</h2>
+<ul>
+<li><a href='settings.html'>Settings</a></li>
+<li><a href='keyboard.html'>Keyboard Shortcuts</a></li>
+<li><a href='read-more.html'>Lychee logic overview</a></li>
+<li><a href='structure.html'>Directory Structure</a></li>
+<li><a href='node.html'>Front-end</a></li>
+<li class='active'><a href='distributions.html'>Distributions Examples</a></li>
+</ul>
+
+                                    <!-- <li>
+                                        <h2>Architecture Concepts</h2>
+                                        <ul>
+                                            <li><a href="docs/lifecycle.html">Request Lifecycle</a></li>
+                                            <li><a href="docs/container.html">Service Container</a></li>
+                                            <li><a href="docs/providers.html">Service Providers</a></li>
+                                            <li><a href="docs/facades.html">Facades</a></li>
+                                            <li><a href="docs/contracts.html">Contracts</a></li>
+                                        </ul>
+                                    </li> -->
+                                    <li class="laravel">
+                                        <h2>Laravel Basics</h2>
+                                        <ul>
+                                            <li><a href="http://laravel.com/docs/7.x/routing">Routing</a></li>
+                                            <li><a href="http://laravel.com/docs/7.x/middleware">Middleware</a></li>
+                                            <li><a href="http://laravel.com/docs/7.x/csrf">CSRF Protection</a></li>
+                                            <li><a href="http://laravel.com/docs/7.x/controllers">Controllers</a></li>
+                                            <li><a href="http://laravel.com/docs/7.x/requests">Requests</a></li>
+                                            <li><a href="http://laravel.com/docs/7.x/responses">Responses</a></li>
+                                            <li><a href="http://laravel.com/docs/7.x/views">Views</a></li>
+                                            <li><a href="http://laravel.com/docs/7.x/urls">URL Generation</a></li>
+                                            <li><a href="http://laravel.com/docs/7.x/session">Session</a></li>
+                                            <li><a href="http://laravel.com/docs/7.x/validation">Validation</a></li>
+                                            <li><a href="http://laravel.com/docs/7.x/errors">Error Handling</a></li>
+                                            <li><a href="http://laravel.com/docs/7.x/logging">Logging</a></li>
+                                        </ul>
+                                    </li>
```

Comment:
I know this is generated, but should this section be commented out? Or, really, should any of these many commented lines be here at all?

Reply:
Nah, I still like to see them because it also explains how the infrastructure works if one wants to dev Lychee. :)

Reply:
It makes more sense in context :) But the others? There's a lot of commented out html there....

Reply:
I was too lazy to completely remove the rest.